src/components/sbb-menu-action/sbb-menu-action.e2e.ts

```diff
@@ -6,42 +6,31 @@ describe('sbb-menu-action', () => {
   beforeEach(async () => {
     page = await newE2EPage();
     await page.setContent('<sbb-menu-action id="focus-id">Menu Action</sbb-menu-action>');
-
     element = await page.find('sbb-menu-action');
   });
 
   describe('events', () => {
     it('dispatches event on click', async () => {
       await page.waitForChanges();
-      const menuAction = await page.find('sbb-menu-action');
       const changeSpy = await page.spyOnEvent('click');
 
-      await menuAction.click();
+      await element.click();
       expect(changeSpy).toHaveReceivedEventTimes(1);
     });
 
+    /*
+    TODO: Figure out why this test fails, when sbb-button test works
```

Comment:
```suggestion
    it('should not dispatch event on click if disabled', async () => {
      element.setAttribute('disabled', true);
      await page.waitForChanges();
      
      const clickSpy = await page.spyOnEvent('click');
      
      element.triggerEvent('click');
      await page.waitForChanges();
      
      expect(clickSpy).not.toHaveReceivedEvent();
    });
```

I suggest to try this approach as the method `triggerEvent()` turns out to be more reliable in cases like this.

Reply:
Thanks for the hint 😃 👍 
I now use `element.triggerEvent('click', { bubbles: true, cancelable: true, composed: true });` and that seems to work.